### ISSUE TITLE
JS API promise never resolves if compilation level is set to BUNDLE

### ISSUE THREAD

**Issue by foobardevpro** [theseanl/tscc#71]
```
tscc({
	modules: {
		'dist/bundle': 'src/main.ts'
	},
	compilerFlags: {
		'compilation_level': 'BUNDLE'
	}
}, 'somepath').then(() => {
	// WILL NOT REACH (BUT WILL COMPILE)
})
```

Will resolve for every other compilation level.

**Comment by theseanl** [comment]
Hi, thank you for reporting this. It seems that when `--compilation_level=BUNDLE` is used, it does not respect `--json_streams` flag and writes a bundle output directory to a file. I think this is a bug of the compiler, or if this is what it intended  to do (I've never tried BUNDLE mode personally), it is undesirable to use tscc with BUNDLE mode. I just added a fix to handle such cases, hope this fixes the issue on your end too.

**Comment by foobardevpro** [comment]
Issue still persists on my end, even with the new changes.

(I wanted to use bundle mode during development, in hopes of shorter compile times)

**Comment by foobardevpro** [comment]
Issue fixed in 0.4.3
